machine-learning/sentiment-analysis/sentiment-analysis.dig

```diff
@@ -0,0 +1,43 @@
+_export:
+  !include : config/params.yml
+  td:
+    database: ${dbname}
+    engine: hive
+
++prepare:
+  +shuffle:
+    _parallel: true
+
+    +train:
+      td>: queries/shuffle.sql
+      source: movie_review_train
+      create_table: movie_review_train_shuffled
+
+    +test:
+      td>: queries/shuffle.sql
+      source: movie_review_test
+      create_table: movie_review_test_shuffled
+
++train:
```

Comment:
explain briefly what train does in comment.

Reply:
Added 7f3d964